src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/query/GraphQLQuery.java

```diff
@@ -1,8 +1,29 @@
 package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.query;
 
+import org.apache.jena.atlas.json.JsonValue;
+
 import se.liu.ida.hefquin.engine.query.Query;
 
+/**
+ * Represents a query for GraphQL
+ */
 public interface GraphQLQuery extends Query
 {
-	// TODO this interface still needs to be defined
+    /**
+     * Adds @param fieldPath to the query.
+     * Fieldpath includes aliasing and subfields are seperated by '/'
+     */
+    public void addFieldPath(String fieldPath);
```

Comment:
This method and `addParameter` imply that an object with this interface is mutable. In general, we define inmutable objects. Thus, it will be best if you just create constuctors in the implementing class that receive the fields and parameters.